### PR TITLE
feat: Support fetching accepted Activity Center notifications by multiple types

### PR DESCRIPTION
**Pull request by ilmotta**
## Summary

- [x] Changes endpoint `ActivityCenterNotificationsBy` to support fetching multiple types of notification in a single query.
- [x] Adds endpoint `UnreadAndAcceptedActivityCenterNotificationsCount` to allow the mobile client to fetch the count of unread & accepted notifications.
- [x] Add `golangci-lint` to Nix shell. This was possible since PR https://github.com/status-im/status-go/pull/3087 was merged.

#### Notes

- If you'd like to understand *why* these changes are needed, please see the [mobile PR](https://github.com/status-im/status-mobile/pull/14785), or issue https://github.com/status-im/status-mobile/issues/14712
- All changes should be completely backwards compatible, and there should be no impact for the desktop app.
- The mobile client has been already tested using this branch.

### CODE REVIEW

**Comment by ghost** [comment]
### Pull Request Checklist
- [x] Have you updated the documentation, if impacted (e.g. [docs.status.im](https://status.im/docs/))?
- [x] Have you tested changes with mobile?
- [x] Have you tested changes with desktop?

File: protocol/activity_center_persistence.go [hunk 1]
```diff
@@ -813,35 +816,53 @@ func (db sqlitePersistence) MarkActivityCenterNotificationsUnread(ids []types.He
...
-    a.reply_message,
-    a.contact_verification_status,
-    c.name,
-    a.author
-    FROM activity_center_notifications a
-    LEFT JOIN chats c
-    ON
-    c.id = a.chat_id
-    WHERE NOT dismissed AND NOT a.accepted AND notification_type = ? AND author = ?`, ActivityCenterNotificationTypeContactRequest, contactID)
+		SELECT
```
Comment (MishkaRogachev) [inline protocol/activity_center_persistence.go:845]: Recently i was looking on those 3 selects.  It would be great to refactor common part from them

**Review by MishkaRogachev — APPROVED** [review]
Screenshot 2023-01-16 at 12 46 50

I've tested this changes with status-desktop, everything seems to be working

**Review by cammellos — APPROVED** [review]

**Comment by ilmotta** [comment]
> I've tested this changes with status-desktop, everything seems to be working

Thank you for double-checking @MishkaRogachev!

**Pull request merged**